src/client/components/landingPage.jsx

```diff
@@ -9,6 +9,7 @@ import { getCurrentGameThunk, findRandomGameThunk, updateNameThunk } from '../st
 
 const LandingPage = (props) => {
   const [name, setName] = useState('');
+  // eslint-disable-next-line no-unused-vars
```

Comment:
what's this one for?

Reply:
It's for the line underneath, I think it's for something kwon is using